Realtime/src/commonMain/kotlin/io/github/jan/supacompose/realtime/events/ChannelAction.kt

```diff
@@ -62,30 +69,46 @@ sealed interface ChannelAction {
         override val columns: List<Column>,
         @SerialName("commit_timestamp")
         override val commitTimestamp: Instant,
-    ): ChannelAction
+    ) : ChannelAction, HasOldRecord
 
     @Serializable
     data class Select(
         override val record: JsonObject,
         override val columns: List<Column>,
         @SerialName("commit_timestamp")
         override val commitTimestamp: Instant,
-    ): ChannelAction
+    ) : ChannelAction, HasRecord
 
 }
 
 /**
- * Decodes [ChannelAction.record] as [T] and returns it or null when either [ChannelAction.record] is null or it cannot be decoded as [T]
+ * Decodes [HasRecord.record] as [T] and returns it or null when either [HasRecord.record] is null or it cannot be decoded as [T]
```

Comment:
```suggestion
 * Decodes [HasRecord.record] as [T] and returns it or null when it cannot be decoded as [T]
```
`HasRecord.record` can't be null now